source/reference/oplog-sizing.txt

```diff
@@ -40,55 +41,30 @@ Estimate ``oplog`` Size Needed for Initial Sync
       cluster. If there are multiple shards, the smallest number is the
       minimum ``oplog`` window.
 
-   .. step:: Estimate Copy Rate During Synching
-
-      .. _c2c-step-est-size:
-
-      To gather performance data while synching, start the sync process
-      and monitor how fast data is transferred between clusters.
-      
-      To start syncing, run the :ref:`/start <c2c-api-start>`
-      command.
+   .. step:: Determine mongosync replication lag
 
-      To get the ``copy_rate``:
-      
-      - run the  :ref:`/progress <c2c-api-progress>` command to get
-        ``estimatedCopiedBytes_time01``
-      - wait a second or two
-      - run the  :ref:`/progress <c2c-api-progress>` command to get
-        ``estimatedCopiedBytes_time02``
-      
-      The ``copy_rate`` is:
-
-      .. code-block:: shell
-
-         copy_rate = ( estimatedCopiedBytes_time02 - estimatedCopiedBytes_01) / time_between_requests
-
-   .. step:: Estimate Copy Time
-
-      Estimate the time needed to copy the entire collection. The
-      estimated copy time is:
-      
-      .. code-block:: shell
+      To get the ``lagTimeSeconds`` value, run the  
+      :ref:`/progress <c2c-api-progress>` command. 
+      The lag time is the time in seconds between the 
+      last event applied by ``mongosync`` and time of the current
+      latest event on the source cluster.
 
-         estimatedCopyTime = estimatedTotalBytes / copy_rate
+      It is a measure of how far behind the source cluster mongosync is.
 
    .. step:: Validate ``oplog`` Size
 
-      If the estimated time is larger than the minimum oplog window you
-      must cancel synchronization. Before restarting, make one of the
-      following changes:
+      If the lag time approaches the minimum ``oplog`` window, you
+      should make one of the following changes:
```

Comment:
(imperative mood) 
It's fine to sometimes imply "you". I try to stay away from "you should" in docs, because it's a little ambiguous.

```suggestion
      If the lag time approaches the minimum ``oplog`` window, make one 
      of the following changes:
```

Reply:
Fixed